src/Button/index.js

```diff
@@ -139,6 +139,12 @@ export const StyledButton = styled(
     }
   }
 
+  &:focus {
+    box-shadow: ${({ tokens, bordered, type }) =>
+        bordered && `inset 0 0 0 1px ${tokens.borderColorButton[type]},`}
```

Comment:
Please use a token for borderWidth (everwhere).

Reply:
We don't have created token `borderWidthButton` yet. Which token did you mean?